src/mapping.ts

```diff
@@ -261,6 +279,10 @@ export function handleShelfRepay(call: BorrowCall): void {
 
   pool.totalRepaysCount = pool.totalRepaysCount + 1
   pool.totalRepaysAggregatedAmount = pool.totalRepaysAggregatedAmount.plus(amount)
+  pool.totalDebt = pool.totalDebt.minus(amount)
+  if (poolFromShelf(shelf).senior !== '0x0000000000000000000000000000000000000000') {
+    pool.seniorDebt = pool.seniorDebt.minus(amount)
```

Comment:
this we don't know 🙈the money can also come from the junior tranche. Would leave it out

Reply:
Thanks, good find. Removed it!